tests/test_dimension.py

```diff
@@ -22,8 +22,8 @@ def test_domain_vs_interior(self):
         t = grid.stepping_dim  # noqa
 
         u = TimeFunction(name='u', grid=grid)  # noqa
-        eqs = [Eq(u.forward, u + 1),
-               Eq(u.forward, u.forward + 2, region=INTERIOR)]
+        eqs = [Eq(u.forward, u + 1.),
+               Eq(u.forward, u.forward + 2., region=INTERIOR)]
```

Comment:
Out of curiosity, what did this fix?  If int vs float in this context matters, I need to be more careful in other places, too.

Reply:
It doesn't change anything, some debug leftover due to types issues at some point.